pandas/tests/test_interval.py

```diff
@@ -63,10 +63,79 @@ def test_hash(self):
         # should not raise
         hash(self.interval)
 
-    # def test_math(self):
-    #     expected = Interval(1, 2)
-    #     actual = self.interval + 1
-    #     self.assertEqual(expected, actual)
+    def test_math_add(self):
+        expected = Interval(1, 2)
+        actual = self.interval + 1
+        self.assertEqual(expected, actual)
+
+        expected = Interval(1, 2)
+        actual = 1 + self.interval
+        self.assertEqual(expected, actual)
+
+        actual = self.interval
+        actual += 1
+        self.assertEqual(expected, actual)
+
+        with self.assertRaises(TypeError):
+            self.interval + Interval(1, 2)
+
+    def test_math_sub(self):
+        expected = Interval(-1, 0)
+        actual = self.interval - 1
+        self.assertEqual(expected, actual)
+
+        expected = Interval(-1, 0)
+        actual = 1 - self.interval
+        self.assertEqual(expected, actual)
+
+        actual = self.interval
+        actual -= 1
+        self.assertEqual(expected, actual)
+
+        with self.assertRaises(TypeError):
+            self.interval - Interval(1, 2)
+
+    def test_math_mult(self):
+        expected = Interval(0, 2)
+        actual = self.interval * 2
+        self.assertEqual(expected, actual)
+
+        actual = self.interval
+        actual *= 2
+        self.assertEqual(expected, actual)
+
+        with self.assertRaises(TypeError):
+            self.interval * Interval(1, 2)
+
+    def test_math_div_py3(self):
```

Comment:
OK, that's a bummer.

For now, let's just make sure the unit tests can pass successfully. I would not stress checking the differences between different versions of python, and would write a single test that unambiguously works with most Python 2 and 3 (by avoiding edge cases that distinguish between floor and true division).